src/main/java/org/medicmobile/webapp/mobile/RDToolkitSupport.java

```diff
@@ -0,0 +1,219 @@
+package org.medicmobile.webapp.mobile;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.os.ParcelFileDescriptor;
+import android.util.Base64;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.rdtoolkit.support.interop.RdtIntentBuilder;
+import org.rdtoolkit.support.interop.RdtUtils;
+import org.rdtoolkit.support.model.session.ProvisionMode;
+import org.rdtoolkit.support.model.session.TestSession;
+import org.rdtoolkit.support.model.session.TestSession.TestResult;
+
+import java.io.ByteArrayOutputStream;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map;
+
+import static android.app.Activity.RESULT_OK;
+import static org.medicmobile.webapp.mobile.EmbeddedBrowserActivity.RDTOOLKIT_CAPTURE_ACTIVITY_REQUEST_CODE;
+import static org.medicmobile.webapp.mobile.EmbeddedBrowserActivity.RDTOOLKIT_PROVISION_ACTIVITY_REQUEST_CODE;
+import static org.medicmobile.webapp.mobile.JavascriptUtils.safeFormat;
+import static org.medicmobile.webapp.mobile.MedicLog.error;
+import static org.medicmobile.webapp.mobile.MedicLog.log;
+import static org.medicmobile.webapp.mobile.Utils.getISODate;
+import static org.medicmobile.webapp.mobile.Utils.json;
+
+public class RDToolkitSupport {
+
+	private final Activity ctx;
+
+	RDToolkitSupport(Activity ctx) {
+		this.ctx = ctx;
+	}
+
+	String process(int requestCode, int resultCode, Intent intentData) {
+
+		log(this, "RDToolkitSupport :: process requestCode=%s", requestCode);
```

Comment:
Should this be debug? (this applies to the other messages that this file logs.

Reply:
I've changed them by trace()